.github/pull_request_template.md

```diff
@@ -20,7 +20,7 @@ Mark an `[x]` for completed items.
   * **PATCH version (last number)**: changes/bug fixes that do not affect
     the api/interface. Examples: typo fix, change/add warning messages...
 * [ ] All inputs are used in process.
-* [ ] All output fields have their ``re-save`` calls.
+* [ ] All output fields have a value assigned to them.
```

Comment:
This is not 100 % exact because some output fields do not have mandatory "outputs". Perhaps something along the lines of 

> All required output fields have a value assigned to them.

On the other hand, this is something we would want to check with a test anyway, right?